### PR TITLE
Filter CCDA export for date range

### LINKED ISSUE #5497: Need to support CCDA export date filtering.
While we support selecting documents to be rendered and sent by date range, we need to filter content matching selected dates.

This filtering will be any encounter and encounter forms within filter dates.
All other section will populate appropriate to the document type.

g(9)(i)(B) Functional Requirements - Specific Date and Date Range
Verified API routine(s) responded to a request for patient data for a date range and that the patient data returned is accurate and without omission based upon developer’s documentation for data return based upon a date range request

### PR DESCRIPTION
**Pull request by sjpadgett**
- mod ccda socket write for better control
- bump db version from prior PR

Fixes #5497

### CODE REVIEW

File: interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php [hunk 1]
```diff
@@ -142,14 +200,18 @@ public function getPatientdata($pid, $encounter)
...
-            <postalCode>" . xmlEscape($result['postal_code']) . "</postalCode>
-            <country>" . xmlEscape($result['country_code']) . "</country>
+            <use>" . xmlEscape('HP') . "</use>
+            <street>" . xmlEscape($result['street'] ?? '') . "</street>
+            <street>" . xmlEscape($result['street_line_2'] ?? '') . "</street>
+            <city>" . xmlEscape($result['city'] ?? '') . "</city>
+            <state>" . xmlEscape($result['state'] ?? '') . "</state>
+            <postalCode>" . xmlEscape($result['postal_code'] ?? '') . "</postalCode>
+            <country>" . xmlEscape($result['country_code'] ?? '') . "</country>
+            " . $previous_addresses . "
```
Comment (adunsulag) [inline interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php:214]: All of the previous address stuff looks great @sjpadgett.  Good job!

Comment (sjpadgett) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php:214]: Thanks, I'm working hard to get a good upcoming performance review from Brady and hopefully a long overdue raise!:)
Anyway, I love our newest human readable HTML CCDA. A very fine job Stephen.
It's almost looks like we know what we're doing!:)

Comment (bradymiller) [reply interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php:215]: performance review 110%
yearly salary raise x2 :)

**Comment by adunsulag** [comment]
Bringing this in, there's a couple problems on the api side of things but I'll fix those in a separate PR.

**Pull request merged**